### PR TITLE
cREAL on contractkit

### LINKED ISSUE #8688: Extend env-test to support cREAL

### LINKED ISSUE #8723: cREAL support for contractkit

### PR DESCRIPTION
**Pull request by martinvol**
### Description

Makes contrackit aware of new cREAL contracts

### Other changes

-

### Tested

Updated

### Related issues

- Fixes #8723
- closes #8688

### Backwards compatibility

None

### Documentation

Autogenerated

### CODE REVIEW

File: packages/env-tests/src/monorepoRun.ts [hunk 1]
```diff
@@ -39,10 +35,10 @@ function runTests() {
...
     runTransfersTest(context, stableTokensToTest)
     runExchangeTest(context, stableTokensToTest)
-    runOracleTest(context)
-    runReserveTest(context)
-    runAttestationTest(context)
-    runGrandaMentoTest(context, stableTokensToTest)
+    // runOracleTest(context)
+    // runReserveTest(context)
+    // runAttestationTest(context)
+    // runGrandaMentoTest(context, stableTokensToTest)
```
Comment (martinvol) [inline packages/env-tests/src/monorepoRun.ts]:
```suggestion
    runOracleTest(context)
    runReserveTest(context)
    runAttestationTest(context)
    runGrandaMentoTest(context, stableTokensToTest)
```

File: packages/env-tests/src/monorepoRun.ts [hunk 2]
```diff
@@ -3,14 +3,10 @@ import Web3 from 'web3'
...
 import { clearAllFundsToRoot, parseStableTokensList } from './scaffold'
-import { runAttestationTest } from './tests/attestation'
 import { runExchangeTest } from './tests/exchange'
-import { runGrandaMentoTest } from './tests/granda-mento'
-import { runOracleTest } from './tests/oracle'
-import { runReserveTest } from './tests/reserve'
 import { runTransfersTest } from './tests/transfer'
 
-const DEFAULT_TOKENS_TO_TEST = [StableToken.cUSD]
+const DEFAULT_TOKENS_TO_TEST = [StableToken.cREAL]
```
Comment (martinvol) [inline packages/env-tests/src/monorepoRun.ts]:
```suggestion
const DEFAULT_TOKENS_TO_TEST = [StableToken.cUSD]
```

File: packages/env-tests/src/tests/exchange.ts [hunk 3]
```diff
@@ -35,10 +35,12 @@ export function runExchangeTest(context: EnvTestContext, stableTokensToTest: Sta
           `quote selling ${stableToken}`
         )
 
+        // console.log(exchange.address)
```
Comment (martinvol) [inline packages/env-tests/src/tests/exchange.ts]:
```suggestion
```

File: packages/env-tests/src/tests/exchange.ts [hunk 4]
```diff
@@ -39,6 +39,7 @@ export function runExchangeTest(context: EnvTestContext, stableTokensToTest: Sta
           .approve(exchange.address, stableTokenAmountToSell.toString())
           .send()
         await approveTx.waitReceipt()
+        // console.log("approve1")
```
Comment (martinvol) [inline packages/env-tests/src/tests/exchange.ts]:
```suggestion
```

File: packages/env-tests/src/tests/exchange.ts [hunk 5]
```diff
@@ -68,6 +68,7 @@ export function runExchangeTest(context: EnvTestContext, stableTokensToTest: Sta
           .approve(exchange.address, goldAmountToSell.toString())
           .send()
         await approveGoldTx.waitReceipt()
+        // console.log("approve2")
```
Comment (martinvol) [inline packages/env-tests/src/tests/exchange.ts]:
```suggestion
```

File: packages/sdk/contractkit/src/wrappers/StableToken.test.ts [hunk 6]
```diff
@@ -25,6 +25,11 @@ testWithGanache('StableToken Wrapper', async (web3) => {
       name: 'Celo Euro',
       symbol: 'cEUR',
     },
+    [StableToken.cREAL]: {
+      stableToken: StableToken.cREAL,
+      name: 'Celo Brazilian Rean',
```
Comment (nicolasbrugneaux) [inline packages/sdk/contractkit/src/wrappers/StableToken.test.ts]: typo
```suggestion
      name: 'Celo Brazilian Real',
```

**Review by nicolasbrugneaux — CHANGES_REQUESTED** [review]
Looks good aside  the typo

**Review by m-chrzan — APPROVED** [review]
Looks good overall. Is it too late to standardize on just one of cREAL vs cBRL throughout the repo?

**Comment by martinvol** [comment]
It should be cREAL everywhere but in the contracts I left the StableTokenBRL (so that it can be directly mapped by the oracles -> CELO/BRL -> StableTokenBRL)

**Review by nicolasbrugneaux — APPROVED** [review]

**Pull request merged**